build.gradle

```diff
@@ -73,7 +74,7 @@ def unpickMetaFile = file("unpick-definitions/unpick.json")
 
 dependencies {
 	enigmaRuntime "cuchaz:enigma-swing:${project.enigma_version}:all"
-	enigmaRuntime "net.fabricmc:stitch:${project.stitch_version}"
```

Comment:
Beyonds the merging and name proposal, another thing we need to take care of is stitch's intermediary generation abilities.

Reply:
Nvm this is enigma than buildscript dependency. Ignore my last comment.